lib/signaling/v2/localparticipant.js

```diff
@@ -249,8 +249,48 @@ class LocalParticipantV2 extends LocalParticipantSignaling {
   setNetworkQualityConfiguration(networkQualityConfiguration) {
     this.networkQualityConfiguration.update(networkQualityConfiguration);
   }
+
+  // lib/signaling/v2/localparticipant.js
+  setPublisherHints(hints) {
+    //     "hints": [
+    //       {
+    //         "encodings": [
+    //           {
+    //             "enabled": true,
+    //             "layer_index": 0,
+    //             "max_bitrate": 500000,
+    //             "max_framerate": 5,
+    //             "render_dimensions": {
+    //               "height": 180,
+    //               "width": 320
+    //             }
+    //           },
+    //           {
+    //             "enabled": false,
+    //             "layer_index": 1
+    //           }
+    //         ],
+    //         "track": "MT123"
+    //       }
+    //     ],
+    hints.forEach(({ track, encodings }) => {
+      const trackSignaling = Array.from(this.tracks.values()).find(trackPub => trackPub.sid === track);
+      if (trackSignaling) {
+        const rtpSender = Array.from(trackSignaling.trackTransceiver._senders)[0];
+        const parameters = rtpSender.getParameters();
+        encodings.forEach(({ enabled, layer_index: layerIndex }) => {
+          parameters.encodings[layerIndex].active = enabled;
+        });
+        rtpSender.setParameters(parameters);
+      } else {
+        // eslint-disable-next-line no-console
+        console.error('track signaling not found: ', track);
```

Comment:
Please use the custom logger here.

Reply:
This class is not yet hooked with custom logger. I will address this in next PR.

Reply:
okay, now its hooked up.